doc/mloginfo.rst

```diff
@@ -393,11 +396,106 @@ For example:
    invoice-prod.invoices     insert       12768411     22233323        86313                12344
    invoice-prod.invoices     insert       12868411     22233323        86313                12344
 
+Sharding (``--sharding``)
+-----------------------------------------
+
+Outputs information of sharding configuration. Includes groupings of
+similar error/warning messages that are specific to sharding. A breakdown
+of chunk migration activity is included, specifying the source/destination
+shard, the number of chunk migrations attempted within the hour, and the
+ratio of successful to failed migrations. Failed chunk migrations specify
+when the failure occurred in formatted as [hours:minutes:seconds:milliseconds]
+but also notes if it precedes to be successful in the future (within the hour
+time frame). It also outputs chunk split statistics, which has a similar output
+as the chunk migration tables.
+
+For example:
+
+.. code-block:: bash
+
+   mloginfo mongod.log --sharding
+
+In addition to the default information, this command also outputs sharding
```

Comment:
This appears to repeat most of the description above (starting from line 402). I would make this more concise and just show the example output.

You could also move some of the detail above (like description of the timestamp format) as further annotation for the example output.

Reply:
I made the outline more succinct